isis/src/docsys/documents/InstallGuide/InstallGuide.xml

```diff
@@ -34,43 +34,70 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
       <li>After the installation has finished, open up a bash prompt in your terminal window.</li>
-      <li>Create a new environment for your ISIS3 installation:
+       <li>If you are running Mac OS X, a pkg file (which looks similar to Anaconda3-5.3.0-MacOSX-x86_64.pkg) will be downloaded.  Double-click on the file to start the installation process.</li>
+      <li>Again from within a terminal window, create a new environment for your ISIS3 installation:
         <pre>
-          #Add the following channels to the Anaconda installation
-          conda config --add channels conda-forge, usgs-astrogeology
-          
+          <em>#Add the following channels to the Anaconda installation</em>
+          <strong>conda config --add channels conda-forge</strong>
+          <strong>conda config --add channels usgs-astrogeology</strong>
+
+          #Verify you have the correct channels:
+          <strong>conda config --show channels</strong>
+
+          <em>#You should see:</em>
+
+          <strong>channels:</strong>
+           <strong>   - usgs-astrogeology</strong>
+           <strong>   - conda-forge</strong>
+           <strong>   - defaults</strong>
+
+           <em>#The order is important.  If conda-forge is before usgs-astrogeology, you will need to run:</em>
+
+           <strong>conda config --add channels usgs-astrogeology</strong>
 
-          #Create a new conda environment to install ISIS3 in
-          conda create -n isis3
 
-          #Activate the environment
-          source activate isis3
+          <em>#Create a new conda environment to install ISIS3 in.</em>
+          <strong>conda create -n isis3</strong>
 
-          #Download the ISIS3 version
-          conda install -c usgs-astrogeology isis3
+          <em>#Activate the environment</em>
+          <strong>source activate isis3</strong>
+
+          <em>#Download the ISIS3 version</em>
+          <strong>conda install -c usgs-astrogeology isis3</strong>
           
-          #Execute the ISIS3 variable initialization script with default arguments.
-          #This script prepares default values for:  $ISISROOT/$ISIS3DATA/$ISIS3TESTDATA
+          <em>#Execute the ISIS3 variable initialization script with default arguments.</em>
+          <em>#This script prepares default values for:  $ISISROOT/$ISIS3DATA/$ISIS3TESTDATA</em>
 
-          python $CONDA_PREFIX/scripts/isis3VarInit.py
+          <strong>python $CONDA_PREFIX/scripts/isis3VarInit.py</strong>
 
          Executing this script with no arguments will result in $ISIS3DATA=$CONDA_PREFIX/data, 
          and $ISIS3TESTDATA=$CONDA_PREFIX/testdata.  The user can specify different directories
          for both of these optional values:
 
-         ./$CONDA_PREFIX/scripts/isis3VarInit --data-dir=[path to data directory]  --test-dir=[path to test data directory]
+         <strong>./$CONDA_PREFIX/scripts/isis3VarInit --data-dir=[path to data directory]  --test-dir=[path to test data directory]</strong>
```

Comment:
Do we need to include the rsync command to get the ISIS3Data?

Reply:
:+1:

Reply:
Oh oops I did not look at the rest of the document, my bad. Looks good.